clang/lib/CodeGen/CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ add_clang_library(clangCodeGen
   PatternInit.cpp
   SanitizerMetadata.cpp
   SwiftCallingConv.cpp
+  LangBuiltins/CGHLSLBuiltins.cpp
```

Comment:
I wouldn't bother with the LangBuiltins subdirectory

Reply:
done!